features/FEATURE_LWIP/lwip-interface/lwip/src/netif/ppp/lwip_utils.c

```diff
@@ -605,7 +605,11 @@ void ppp_print_string(const u_char *p, int len, void (*printer) (void *, const c
  * ppp_logit - does the hard work for fatal et al.
  */
 static void ppp_logit(int level, const char *fmt, va_list args) {
+#ifdef PPP_LOGIT_BUFSIZE
+    char buf[PPP_LOGIT_BUFSIZE];
+#else
```

Comment:
This is not redundant. Its there for the default case, i.e., PPP_LOGIT_BUFSIZ is not defined. For example when using Ethernet you will have PPP_LOGIT_BUFSIZ = 1024B